plutus-pab/app/Cli.hs

```diff
@@ -140,11 +140,12 @@ runConfigCommand ::
     -> Configuration -- ^ Monitoring configuration
     -> Config        -- ^ PAB Configuration
     -> Availability  -- ^ Token for signaling service availability
+    -> App.EventfulBackend -- ^ Whether to use the sqlite or the in-memory backend
```

Comment:
Seeing how the number of arguments keeps on increasing and the respective arguments dealing with different levels of abstraction it might be good to consider some refactoring(s).

- The `Configuration` type comes from some external dependency iirc but the completely generic name isn't great
- `Config` vs `Configuration` is confusing. So maybe `PabConfig` and `MonitoringConfig` or so
- I frankly don't recall how the `Availability` token is being used
- The new eventful environment could be part of the PAB Config I guess (?)

Under different circumstances i would even put this in a Reader monad but i guess we don't want to add that into the mix of what's already there :sweat_smile:

Reply:
I can make a record with fields for all of the parameters